pages/guides/how-to-create-tradingview-charts.mdx

```diff
@@ -110,3 +110,15 @@ The TradingView integration is a web-based widget that allows users to view Pyth
 5. Replace the `theme` parameter with the theme you want to display. For example, to display the price of Ethereum in dark mode, use `theme: "dark"`.
 
 6. There is a fully working open-source example of the TradingView integration by one of Pyth's contributors [here](https://github.com/cctdaniel/pyth-tv-example). The example application is deployed [here](https://pyth-tv-example.vercel.app/).
+
+## Using Datafeed URL with Charting Library
+
+We also provide a datafeed URL that follows the [TradingView Data Integration spec](https://www.tradingview.com/rest-api-spec/#tag/Data-Integration). You can implement your own datafeed utilizing the API or use the built-in [UDF](https://www.tradingview.com/charting-library-docs/latest/connecting_data/UDF) adapter. If you need a step-by-step guide, refer to the [How to connect data via Datafeed API](https://www.tradingview.com/charting-library-docs/latest/tutorials/implement_datafeed_tutorial/) tutorial.
+
+The datafeed URL is [here](https://benchmarks.pyth.network/v1/shims/tradingview) and documentation can be found [here](https://benchmarks.pyth.network/redoc)
+
+### Example
```

Comment:
I read the guides linked and they seemed a little complicated. I think the step-by-step instructions in "Technical Integration" are great. Can we do something similar for this version of connecting to tradingview?

Or should we write some code to make this easy for users? Maybe we can just ask someone who's already done this for their code and share that here?